documentation/src/components/PageNavigation.astro

```diff
@@ -0,0 +1,97 @@
+---
+import Show from 'src/components/Show.astro'
+import { getPathParamsFromId } from 'src/languages'
+import { getLocalizedSidebar } from 'src/navigation'
+import { getPrevNext } from 'src/navigation'
+import {PAGE_NAVIGATION} from 'src/consts'
+
+const { slug, lang } = getPathParamsFromId(Astro.url.pathname);
+const baseSidebar = await getLocalizedSidebar(slug, lang);
+
+const { prevPage, nextPage } = getPrevNext(baseSidebar, Astro.url.pathname);
+---
+
+<nav class="pagination-links">
+  <Show when={prevPage}>
+    <a href={prevPage?.link} rel="prev" class="pagination-link pagination-link-prev">
+      <svg aria-hidden="true" class="pagination-link-icon" width="16" height="16" viewBox="0 0 24 24" fill="currentColor"><path d="M17 11H9.41l3.3-3.29a1.004 1.004 0 1 0-1.42-1.42l-5 5a1 1 0 0 0-.21.33 1 1 0 0 0 0 .76 1 1 0 0 0 .21.33l5 5a1.002 1.002 0 0 0 1.639-.325 1 1 0 0 0-.219-1.095L9.41 13H17a1 1 0 0 0 0-2Z"></path></svg>
```

Comment:
It would be cool to move this icons into https://github.com/effector/effector/tree/master/documentation/src/icons

Reply:
resolved!